src/documents/management/commands/document_importer.py

```diff
@@ -72,11 +72,24 @@ def handle(self, *args, **options):
         if not os.access(self.source, os.R_OK):
             raise CommandError("That path doesn't appear to be readable")
 
-        manifest_path = os.path.normpath(os.path.join(self.source, "manifest.json"))
-        self._check_manifest_exists(manifest_path)
+        manifest_paths = []
 
-        with open(manifest_path) as f:
+        main_manifest_path = os.path.normpath(
+            os.path.join(self.source, "manifest.json"),
+        )
+        self._check_manifest_exists(main_manifest_path)
+
+        with open(main_manifest_path) as f:
             self.manifest = json.load(f)
+        manifest_paths.append(main_manifest_path)
+
+        for root, dirs, files in os.walk(self.source):
+            for file in files:
+                if file.endswith("-manifest.json"):
```

Comment:
Pathlib globbing should be a quicker way to locate `*.json` and perhaps even `*-manifest.json`.  It would be something like `.glob(**/*-manifest.json)`

Reply:
fix by ead820eeb62b33872a3f4eaa63a6cafc7373e4dc